src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,45 +1,98 @@
-import { useState } from 'react';
+import { FormEvent, useState } from 'react';
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
 
-export const NewMovie = () => {
+interface Props {
+  onAdd: (movie: Movie) => void;
+}
+
+export const NewMovie: React.FC<Props> = ({ onAdd }) => {
   // Increase the count after successful form submission
   // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+  const [count, setCount] = useState(0);
+  const [title, setTitle] = useState('');
+  const [description, setDescription] = useState('');
+  const [imgUrl, setImageURL] = useState('');
+  const [imdbUrl, setImdbURL] = useState('');
+  const [imdbId, setImdbID] = useState('');
+
+  const isValid = () => {
```

Comment:
```suggestion
  const isDisabled = () => {
```
або
```suggestion
  const isInvalid = () => {
```
isValid звучить так що має поветрати тру, коли все заповнено

Reply:
виправ це